test/summarize.test.js

```diff
@@ -0,0 +1,23 @@
+const fs = require('fs').promises;
+const summarize = require('../summarizer/summarize');
+
+
+describe('summarize tests...', () => {
+  let inputData;
+  let predicted;
+  let expected;
+
+  beforeAll(async () => {
+    inputData = await fs.readFile('./summarizer/input.txt', 'utf-8');
+    expected = await fs.readFile('./summarizer/expected.txt', 'utf-8');
+  });
+
+  test('summarize returns expected value', async () => {
+    predicted = await summarize(inputData, 5);
+    return expect(predicted).toBe(expected);
+  });
+
+  test('summarize returns string', async () => expect(typeof predicted).toBe('string'));
```

Comment:
Tests can be run in different order, so don't count on `predicted` to exist here.  I'd re-calculate this in each test so you know it's good.  Try not to have `test()` functions depend on shared state.  It can lead to random failures.